src/ossf_scorecard/scorecard.py

```diff
@@ -0,0 +1,23 @@
+import requests
+
+
+def get_scorecard(platform, org, repo):
+    url = f"https://api.securityscorecards.dev/projects/{platform}/{org}/{repo}"
+    response = requests.get(url)
+
+    if response.status_code == 200:
```

Comment:
Can there be timeouts or other status codes, is there any api rate limiting here?

Reply:
have given other status code checks also 

https://github.com/nexB/ScoreCode/blob/60a880f7182feb32843ba0b964c05987edd5eebe/src/ossf_scorecard/scorecard.py#L26